### PR TITLE
Added Json File for LinkFree

### PR DESCRIPTION
**Pull request by sanjay-kv**
Json File added for Sanjay 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/408"></a>

### CODE REVIEW

File: public/data/sanjay-kv.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      "description": "Became a GitHub Star with 30 other people",
+      "url": "http://github.com/eddiejaoude"
+    },
+    {
+      "title": "GitHub Star of the Year",
+      "date": "2020",
+      "icon": "github",
+      "color": "green",
+      "description": "Won GitHub Star of the Year out of 55+ million people",
+      "url": "http://github.com/eddiejaoude"
```
Comment (eddiejaoude) [inline public/data/sanjay-kv.json:54]: This is not your data?

File: package.json [hunk 2]
```diff
@@ -65,4 +65,4 @@
   "cypress-cucumber-preprocessor": {
     "nonGlobalStepDefinitions": true
   }
-}
\ No newline at end of file
+}
```
Comment (eddiejaoude) [inline package.json:68]: These file changes are not required

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Includes example data

File: public/data/sanjay-kv.json (hunk 1 above)
Comment (sanjay-kv) [reply public/data/sanjay-kv.json:54]: oops. thats a mistake. i forgot to add that changes to commit

**Comment by sanjay-kv** [comment]
All the changes requested has been completed and requested new PR

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please see the comments above.

File: package.json (hunk 2 above)
Comment (krishguptadev) [inline package.json:68]:
```suggestion
}

```

File: public/data/sanjay-kv.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+    {
+      "title": "Stack developer SEO Consultant - Intern",
+      "date": "June 2018",
+      "icon": "google",
+      "color": "green",
+      "description": "Improved SEO results on Google search and Bing",
+      "url": "https://www.linkedin.com/in/sanjay-k-v/"
+    
+}
```
Comment (krishguptadev) [inline public/data/sanjay-kv.json:80]: Please format this file

**Review by krishguptadev — CHANGES_REQUESTED** [review]

**Comment by sanjay-kv** [comment]
Made the necessary changes.

File: public/data/sanjay-kv.json [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    {
+      "title": "Software Engineer",
+      "date": "2018",
+      "icon": "google",
+      "color": "green",
+      "description": "Software Engineer at L&T Infotech",
+      "url": "https://crowdsource.google.com/about/blog/community-spotlight-friendship/"
+    }
+  ]
+
```
Comment (krishguptadev) [inline public/data/sanjay-kv.json:64]:
```suggestion
```
Also add a new line at the bottom of the file

**Review by krishguptadev — CHANGES_REQUESTED** [review]

File: public/data/sanjay-kv.json [hunk 5]
```diff
@@ -61,5 +60,4 @@
       "url": "https://crowdsource.google.com/about/blog/community-spotlight-friendship/"
     }
   ]
-
-}
+}
```
Comment (krishguptadev) [inline public/data/sanjay-kv.json:63]:
```suggestion
}

```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (krishguptadev) [reply package.json:68]: @sanjay-kv Please fix this

**Comment by eddiejaoude** [comment]
@sanjay-kv still 1 comment remaining https://github.com/EddieHubCommunity/LinkFree/pull/408#pullrequestreview-769474595 then we can merge

**Review by Panquesito7 — APPROVED** [review]
Just revert the changes in the packages files and LGTM. 👍

**Comment by krishguptadev** [comment]
Why was this closed?

**Pull request closed without merging**